assembl/lib/frontend_urls.py

```diff
@@ -65,7 +65,7 @@ def __init__(self, discussion):
 
     @classmethod
     def register_frontend_routes(cls, config):
-        from assembl.views.backbone.views import home_view
+        from assembl.views.discussion.views import home_view
```

Comment:
Why make the word "react" appear clearly in the architecture and not the word "backbone"? We should probably choose none or both?